dcicutils/qa_utils.py

```diff
@@ -12,17 +12,30 @@
 from .misc_utils import PRINT, ignored, Retry
 
 
+def show_elapsed_time(start, end):
+    """ Helper method for below that is the default - just prints the elapsed time. """
+    print('Elapsed: %s' % (end - start))
```

Comment:
This should use `PRINT` from the above import so that grepping for stray `print` won't find this. The purpose of the uppercase version is to say "this is intended programmatic output part of a published feature", not debugging. Mind you, this published feature is for debugging. But on the metalevel at which things matter, this intends to print.